build.xml

```diff
@@ -11,6 +11,14 @@
       <ant dir="./store"  target="create-version-sql" inheritAll="true"/>
    </target>
 
+   <target name="publish-local">
+      <ant dir="./native" target="publish-local" inheritAll="true"/>
```

Comment:
This is quite similar to a definition in `build-common.xml` which is included in the sub-directory build.xml files.  To prevent duplication, could just do:

        <ant dir="./store" target="publish-foss-local" inheritAll="true"/>

Or alternatively, make that task use your new task?

Could also do something similar with the build-common.xml task "deploy-foss"